test/eradius_client_SUITE.erl

```diff
@@ -37,7 +37,7 @@ all() -> [
 init_per_suite(Config) ->
     {ok, _} = application:ensure_all_started(eradius),
     startSocketCounter(),
-    application:set_env(lager, handlers, [{lager_journald_backend, []}]),
+    %% application:set_env(lager, handlers, [{lager_journald_backend, []}]),
```

Comment:
Since lager was removed maybe we should remove this line as well instead of commenting it?